Inferential Stats/racial_disc/sliderule_dsi_inferential_statistics_exercise_2_Michael_Skrzypiec.ipynb

```diff
@@ -0,0 +1,471 @@
U35Gw4YNsx555BHHnxcuXGiF\nhYVZQ4YMcarl4YcftoYPH/6HYztVTSfz448/WgkJCVZISIg1YMAA69NPP7XuvPNO69VXX3V5bJZl\nWV9++aU1ZMgQx3ZOnMffH3+/32ZVVZX1yiuvWDfccIPj8/voo48c6//8889WYGCgtWHDBqfae/Xq\nZc2fP/+U46uqqrKioqKsMWPG1Oj76quvrDvuuMMKCwuzevToYT3yyCNWYWHhSbczYsQIa9CgQX+4\nn88//9wKDAy0li9fblmWZW3dutW66667rJCQEOuGG26wXn31VcffV78/ni3Lsr799ltryJAhVnBw\nsDVo0CBr48aNVp8+fayMjAzHHD7wwANWZGSk1bNnT+uee+6xvv32WyskJMR65513LMuqeQwB5xM3\ny2rYr3i47bbb1Lt3bz388MPKy8vT8OHD9dprr2nhwoXy8vLS008/rdzcXI0ePVqvv/66QkJClJaW\npqVLlzpelDxmzBgNHDhQ9957r3bu3KmEhAQtWrRIXbp00YwZM/Trr786XqoNAOe78vJy3XDDDXro\noYcctw4AQGPQoPeIbt26Vb/99pv+9re/qUmTJgoICNB7772nSy+9VFlZWZo4caLjnXNxcXGOb9jI\nyMjQyJEj5efnJz8/P40dO1bLly+XdPydev3791dwcLA8PDw0efJkrV+/XkVFRQ05NACoc+Xl5Xr5\n5Zc1ZswYVVZWOt4lCgCNRYMG0e3bt6tz5876f//v/6lv377685//rO+++04HDx6UzWZzeiWFv7+/\n7Ha7JMlut6tz585OfdVP8trtdqenfVu2bCkfHx/HugBwvrLZbFq+fLnsdrteeOEFp3sHAaAxaNCH\nlQ4ePKiNGzeqd+/e+vLLL/X9999r9OjRmj9/fo33xHl6eqq0tFTS8RuyT/wfsKenp6qqqlRWVqaS\nkhJ5eXk5revl5eVYFwDOZ59//rnpEgCg3jRoEPXw8FDLli01evRoScdfj3LjjTfq5ZdfrvENEaWl\npY5XjJwYSqv73N3d5eHhUaNPOh5cq9c9nf379+vAgQNObZWVlTp27Ji6dOlSq/cFAgAAwHUNmrL8\n/f1VUVEhy7KcXqh89dVXKycnRwUFBWrbtq2k49+vW33JPSAgQHl5eY4XVJ94Ob66r1pRUZGKi4ud\nLtefSlpa2h++DiUrK8vpRcsAAACoOw16j2ifPn3k5eWluXPnqrKyUps3b9Znn32mm2++WdHR0UpJ\nSVFpaam2bdumzMxMxcfHSzr+Muc33nhDe/fu1b59+5SamqrBgwdLkmJjY7V69Wpt3rxZx44d06xZ\nsxQVFeXy+/ASEhK0cuVKp58333yzvqYAAAAA/78Gf33T7t279dRTT+n7779X8+bNNWHCBA0ePFgH\nDx7Uk08+qezsbHl7e2vChAkaMmSIpONnTV9++WUtW7ZM5eXlGjRokKZOneo4q7py5UrNnj1bhYWF\nioyM1MyZM8/q+37z8/MVExPDGVEAAIB61OBB9HxAEAUAAKh/fNc8AAAAjCCIAgAAwAiCKAAAAIwg\niAIAAMAIgigAAACMIIgCAADACIIoAAAAjCCIAgAAwAiCKAAAAIwgiAIAAMAIgigAAACMIIgCAADA\nCIIoAAAAjCCIAgAAwIimpgs4l1133XuSfEyXcdYqKn7V4sU36IYbepsuBQCAC8KePXscv19++eUG\nKzm3EURPIT//TlVUtDddRh34UaWle06/GAAAqBO//PKL43eC6B/j0jwAAACMIIgCAADACIIoAAAA\njCCIAgAAwAiCKAAAAIzgqfkLQrl27fpBmze3MF1InQkODpbNZjNdBgAAOAsE0QvCD5o4sZnpIupQ\nrnJypPDwcNOFAACAs0AQvWAESSK4AQCAcwf3iAIAAMAIgigAAACMIIgCAADACIIoAAAAjCCIAgAA\nwAiCKAAAAIwgiAIAAMAIgigAAACMIIgCAADACIIoAAAAjCCIAgAAwAiCKAAAAIwgiAIAAMAIgigA\nAACMIIgCAADACIIoAAAAjCCIAgAAwAiCKAAAAIwgiAIAAMAIgigAAACMIIgCAADACIIoAAAAjCCI\nAgAAwAiCKAAAAIwgiAIAAMAIgigAAACMIIgCAADACIIoAAAAjCCIAgAAwAiCKAAAAIwgiAIAAMAI\ngigAAACMIIgCAADAiAYPogsXLlS3bt0UHh6usLAwhYeHKycnR8XFxfrrX/+qyMhIRUdHa9myZU7r\npaSkqHfv3urZs6dmzpwpy7IcfZmZmerfv7/CwsI0btw4FRYWNvSwAAAAUEsNHkR37NihyZMna/Pm\nzdqyZYs2b96siIgIPf7442revLmys7M1Z84cPf/889q2bZskKS0tTevWrVNmZqZWrFihnJwcLVy4\nUJK0c+dOTZ8+XbNnz9bGjRvVunVrJSUlNfSwAAAAUEsNHkRzc3PVpUsXp7ajR48qKytLEydOlM1m\nU0hIiOLi4pSeni5JysjI0MiRI+Xn5yc/Pz+NHTtWy5cvl/S/s6HBwcHy8PDQ5MmTtX79ehUVFTX0\n0AAAAFALDRpES0tLlZeXp8WLF6tv37665ZZb9MEHH+jnn3+WzWZTu3btHMv6+/vLbrdLkux2uzp3\n7uzUl5eX5+gLCAhw9LVs2VI+Pj6OdQEAAHBuatqQO9u3b58iIiI0fPhw9e7dW999953uv/9+3XPP\nPbrooouclvX09FRpaakkqaSkRJ6enk59VVVVKisrU0lJiby8vJzW9fLycqx7Ovv379eBAwec2goK\nCs5keAAAAKiFBg2i7du311tvveX4c2RkpAYNGqRNmzaprKzMadnS0lI1a9ZMknMore5zd3eXh4dH\njT7peHCtXvd00tLSNHfu3DMdEgAAAM5QgwbRHTt26J///KfGjBnjaDt27Jguv/xyffPNNyooKFDb\ntm0lSXl5eY
```

Comment:
How about confounding variables? Take a look at the original research paper https://www.povertyactionlab.org/sites/default/files/publications/3%20A%20Field%20Experiment%20on%20Labor%20Market%20Discrimination%20Sep%2004.pdf